packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -86,7 +86,8 @@ export default {
     enableRtx: true,
     receiveTranscription: false,
     experimental: {
-      enableMediaNegotiatedEvent: false
+      enableMediaNegotiatedEvent: false,
+      enableUnifiedMeetings: false
```

Comment:
just wondering if this should be under "experimental"... when will it be moved out of "experimental"?

Reply:
We will remove it when its ready and everyone needs to be defaulted